angrmanagement/ui/dialogs/preferences.py

```diff
@@ -172,11 +190,16 @@ def _on_load_scheme_clicked(self):
         self._load_color_scheme(self._schemes_combo.currentText())
         self.save_config()
 
+    def revert_unsaved(self):
```

Comment:
Identical to base class implemantation

Reply:
The base class raises `NotImpelementedError`.